dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/ExtendedNode.java

```diff
@@ -9,19 +9,18 @@
  * An extended node can be one of several things (depending on its {@code type}):
  *
  * <ul>
- *   <li><em>NODE</em>: {@link CFGBuilder.NodeHolder}. An extended node of this type is just a
- *       wrapper for a {@link Node} (that cannot throw exceptions).
- *   <li><em>EXCEPTION_NODE</em>: {@link CFGBuilder.NodeWithExceptionsHolder}. A wrapper for a
- *       {@link Node} which can throw exceptions. It contains a label for every possible exception
- *       type the node might throw.
- *   <li><em>UNCONDITIONAL_JUMP</em>: {@link CFGBuilder.UnconditionalJump}. An unconditional jump to
- *       a label.
- *   <li><em>TWO_TARGET_CONDITIONAL_JUMP</em>: {@link CFGBuilder.ConditionalJump}. A conditional
- *       jump with two targets for both the 'then' and 'else' branch.
```

Comment:
References (`@link`) in these docs were outdated, which tooling complained about as soon as the class was made public. Otherwise this is unchanged.